docs/application/dotnet/guides/device/attached-devices.md

```diff
@@ -215,18 +216,18 @@ To control LEDs on the device:
 
 To lock and unlock the CPU state:
 
--   Lock the power state with the `RequestCpuLock()` method of the [Tizen.System.Power](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.Power.html) class.
+-   Lock the power state with the `RequestLock()` method of the [Tizen.System.Power](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.Power.html) class.
 
-    The method locks the CPU for a specified time. After the given time (in milliseconds), the lock is unlocked. If the process is destroyed, every lock is removed.
+    The method locks the given PowerLock of the [Tizen.System.PowerLock](https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.PowerLock.html) enumeration for a specified time. After the given time (in milliseconds), the lock is unlocked. If the process is destroyed, every lock is removed.
```

Comment:
The https://developer.tizen.org/dev-guide/csapi/api/Tizen.System.PowerLock.html  link is not working.

Following is the error message:

![404_error](https://user-images.githubusercontent.com/35525967/45606390-5ca23480-ba34-11e8-8aac-4a84e2b053c7.PNG)

Reply:
There is a comment about this on http://suprem.sec.samsung.net/jira/browse/TCSACR-168.
I don't need to create Tizen.System.PowerLock.html manually.
--------------------------------------------------------------------------------------------------------------------
Dear Pureum Jung,

You don't need to create or update API reference. Doc team will copy TizenFX API reference from GitHub(https://samsung.github.io/TizenFX/master/) later.

If you want to update the links to URLs of methods, please update the file name.

Reply:
RequestCpuLock and ReleaseCpuLock is deprecated and I made RequestLock and ReleaseLock because it needs to provide lock and release lock for other lock types besides cpu